.github/workflows/ci.yml

```diff
@@ -811,6 +811,12 @@ jobs:
               --benchmark_files="${EXECUTION_BENCHMARK_RESULTS_DIR}/benchmark-results-c2-standard-16.json" \
               --benchmark_files="${EXECUTION_BENCHMARK_RESULTS_DIR}/benchmark-results-a2-highgpu-1g.json" \
               --compile_stats_files="${COMPILE_STATS_RESULTS}"
+      - name: Trigger DB import
+        continue-on-error: true
```

Comment:
Same here, why do we continue with error?

Reply:
The step might fail when the cloud function will fail and I will have to iterate on the cloud function in the next week and would prefer not to break the CI in those cases.

Reply:
I added a TODO